### PR TITLE
Changed Vm name

### PR DESCRIPTION
**Pull request by sshveta**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 1ebb45604cc65d55a228acf81c8d9f419e800424:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/tests/infrastructure/test_vm_migrate.py [hunk 1]
```diff
@@ -24,10 +24,10 @@ def test_vm_migrate(setup_provider, provider, request):
     Metadata:
         test_flag: migrate, provision
     """
-    vm = VM.factory("vmtest", provider)
+    vm = VM.factory("auto_test_services", provider)
     vm.migrate_vm("email@xyz.com", "first", "last")
     flash.assert_no_errors()
-    row_description = 'vmtest'
+    row_description = 'auto_test_services'
```
Comment (psav) [inline cfme/tests/infrastructure/test_vm_migrate.py]: Shouldn't this start with "test" so the reaper sees it and eliminates it if it is left straggling?

Comment (sshveta) [inline cfme/tests/infrastructure/test_vm_migrate.py]: Need an already existing VM to migrate so the vm should not be eliminated.
I am using auto_test_services for few other tests .

Comment (psav) [inline cfme/tests/infrastructure/test_vm_migrate.py]: If not already, please add a comment to explain this.

**Comment by dajoRH** [comment]
Lint report for commit c241a2535a1f6f68dfb7579d1cef78752101aeb4:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request merged**